nova/scheduler/filters/__init__.py

```diff
@@ -49,11 +49,21 @@ def host_passes(self, host_state, filter_properties):
         """
         raise NotImplementedError()
 
+    def host_info_for_instance_ids(self, _):
```

Comment:
Why `_` instead of a properly named variable, so that an implementer of the method does not have to go and find with what it is called?

Given the name of the function, I would assume it return `host_info` and be given a sequence of instance ids. Could we name it differently? `host_info_requiring_instance_ids` maybe?

Reply:
Done